DigitalExcellence/dex-frontend
# Bugfix/440 styling fixes project overview

## Description

While implementing the overview page there were a few minor details that were missed. We implemented the following changes:
- Make card width larger
- Make side menu smaller on big screens
- Resolve blurry pop-up
- Open detail page on collaborator tab when the user clicks collaborators

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Go to the project overview page
2. The styling should be consistent.
3. Change to mobile view.
4. See that the toggle-list-view-button is hidden.
2. Click the collaborators item on one of the projects
3. The detail page should open on the collaborators tab

## Link to issue

Closes: #440
